www/components/core/Search.tsx

```diff
@@ -93,20 +96,41 @@ export const Search = ({ descriptors }: SearchParams) => {
     }
   }, [searchTerm]);
 
+  function keyPressHandler(event: React.KeyboardEvent, index: number) {
+    if (event.key == "ArrowDown" && index + 1 < (results?.length ?? 0)) {
```

Comment:
I'd opt for triple equals here.
```suggestion
    if (event.key === "ArrowDown" && index + 1 < (results?.length ?? 0)) {
```